AmazonChimeSDK/AmazonChimeSDK/session/DefaultMeetingSession.swift

```diff
@@ -49,6 +49,23 @@ import Foundation
         let activeSpeakerDetector =
             DefaultActiveSpeakerDetector(audioClientObserver: audioClientObserver,
                                          selfAttendeeId: configuration.credentials.attendeeId)
+
+        let contentModality = String(DefaultModality.separator) + ModalityType.content.description
```

Comment:
should we call it as `String(describing:)` as it is discouraged as described [here](https://developer.apple.com/documentation/swift/customstringconvertible/1539130-description)?